code/procs/stat_logging.dm

```diff
@@ -163,6 +163,26 @@
 				if (M.master)
 					var/mob/mymaster = ckey_to_mob(M.master)
 					if (mymaster) special = mymaster.real_name
+			if (ROLE_FLOCKMIND)
+				var/relay_successful = FALSE
+				if (isflockmob(M.current))
+					if (!istype(M.current, /mob/living/critter/flock/drone))
+						var/mob/living/intangible/flock/flockmind/flockmind = M.current
+						relay_successful = flockmind.flock.relay_finished
+					else
+						var/mob/living/critter/flock/drone/flockdrone = M.current
+						relay_successful = flockdrone.flock.relay_finished
+				special = "Relay transmission [relay_successful ? "successful" : "unsuccessful"]"
```

Comment:
This is already tracked by an objective.

Reply:
Included for same reason as above